src/controllers/words.js

```diff
@@ -43,6 +43,11 @@ export const searchWordUsingEnglish = async ({ query, searchWord, ...rest }) =>
   return sortDocsBy(searchWord, words, 'definitions[0]');
 };
 
+function getWordSearchFunction(searchWord) {
+  if (searchWord.match(/".*"/) || searchWord.match(/'.*'/)) { return searchWordUsingEnglish; }
```

Comment:
Conditional blocks shouldn't be on the same line as the conditional statement. It should look something like this:
```
if (conditionalStatement) {
    return conditionalLogic;
}
```
instead of:
```
if (conditionalStatement) { return conditionalLogic; } 
```

If you want to achieve a single line function, I recommend using ternary operators:

```
const getWordSearchFunction = (searchWord) => (
    conditionalStatement ? searchWordUsingEnglish : searchWordUsingIgbo
)
```